Twig/RollbarExtension.php

```diff
@@ -128,8 +128,24 @@ function isFromAllowedHosts(filename) {
         return false;
     }
 
+    function isStringError(payload) {
+        try {
+            if (payload.data.body.message.body !== undefined) {
+                return true;
+            }
+        } catch (e) {
+        }
+
+        return false;
+    }
+
     function ignoreRemoteUncaught(isUncaught, args, payload) {
         try {
+            //this prevents breaking simple string reporting
+            if (isStringError(payload)) {
```

Comment:
not sure if we need to introduce a separate funciton here, it's used only once, it's basically one line + try-catch block